### PR TITLE
HWKAPM-645 : Fix Top navigation to match Patternfly

### PR DESCRIPTION
**Pull request by ammendonca**

### CODE REVIEW

**Comment by ammendonca** [comment]
image

@objectiser, please review.

@cardosogabriel, for some reason simply applying the same styles as in patternfly examples doesn't seem to work (image was expanded, not padded, etc) so had to "overwrite" with inline styles. If you can check/fix what's wrong. Thanks.

**Comment by objectiser** [comment]
@ammendonca LGTM :+1: Thanks!

Will wait for @cardosogabriel to comment before merging.

File: ui/src/main/webapp/index.html [hunk 1]
```diff
@@ -140,6 +140,11 @@
 
   <body>
     <nav class="navbar navbar-default navbar-fixed-top navbar-pf" role="navigation">
+      <div class="navbar-header">
+        <a class="navbar-brand" href="/hawkular-ui/apm" style="padding: 12px;">
+          <img src="img/hawkular-apm-brand.svg" alt="Hawkular Application Performance Management" style="height: 9.1px;">
```
Comment (cardosogabriel) [inline ui/src/main/webapp/index.html]: If needed, you can set the height to 10px. That's what patternfly uses.

**Comment by ammendonca** [comment]
Fixed alignment and size

image

**Comment by objectiser** [comment]
@ammendonca @cardosogabriel Thanks!

**Pull request merged**